source/1_engine/rex_engine/src/filesystem/path.cpp

```diff
@@ -0,0 +1,576 @@
+#include "rex_engine/filesystem/path.h"
+#include "rex_std_extra/string.h"
+#include "rex_std/algorithm.h"
+#include "rex_std/bonus/platform.h"
+#include "rex_engine/diagnostics/win/win_call.h"
+#include "rex_engine/numeric.h"
+#include "rex_std_extra/time/win/win_time_functions.h"
+
+#include <random>
+#define NOMINMAX
+#include <Windows.h>
+#include <filesystem>
+
+namespace rex
+{
+  namespace path
+  {
+    constexpr char8 g_seperation_char = '/';
```

Comment:
Is this the case for all platforms, or is our virtual file system converting these to the correct ones?

Reply:
'/' can be used for all platforms